python/hail/expr/functions.py

```diff
@@ -2040,76 +2146,6 @@ def is_strand_ambiguous(ref, alt) -> BooleanExpression:
     return alleles.contains((ref, alt))
 
 
-@typecheck(ref=expr_str, alt=expr_str)
-def allele_type(ref, alt)-> StringExpression:
```

Comment:
was the reorg intentional? It makes it harder to see/track changes, so imo we shouldn't do that unless necessary.

Reply:
I thought python stuff had to be defined top down, so I moved `allele_type` and `hamming` above the other alt allele methods so I could use them. Although looking at what I ended up doing I didn't actually need to move allele_type since I defined an enumerated version instead, so I'll move that back down.

Reply:
it's top down per scope, but you can do:
```
def f():
    return a + 5
a = 2
```

Reply:
Ah, right. I'll move the other one back as well, then